protocol/x/clob/keeper/mev.go

```diff
@@ -773,7 +773,10 @@ func (k Keeper) AddSettlementForPositionDelta(
 			// Add the settlement to the subaccount.
 			// Note: Funding payment is the negative of settlement, i.e. positive settlement is equivalent
 			// to a negative funding payment (position received funding payment) and vice versa.
-			cumulativePnL.AddDeltaToSubaccount(subaccountId, bigNetSettlement)
+			cumulativePnL.AddDeltaToSubaccount(
+				subaccountId,
+				bigNetSettlementPpm.Div(bigNetSettlementPpm, lib.BigIntOneMillion()),
```

Comment:
To confirm, the rounding behavior here doesn't change in this PR right? We are still rounding the settlement per `perpetualId`, per subaccoujt

In subaccount keeper below, we are rounding only after summing settlement for all perpetual position of a subaccount

I think this is fine, but defer to @jayy04 to confirm

Reply:
yeah
* for mev, behavior isn't changed
* for subaccounts, division is done after summing all perpetual positions of a subaccount